packages/components/src/components/Link/Link.test.tsx

```diff
@@ -424,28 +411,30 @@ describe('Link', () => {
     })
 
     it('should respect removing the icon', () => {
-      component = render(<Link {...iconOverrideProps} icon={'no icon'} />)
-      const icon = component.UNSAFE_queryByType(Icon)
+      render(<Link {...iconOverrideProps} icon={'no icon'} />)
+      const icon = screen.UNSAFE_queryByType(Icon)
 
       expect(icon).toBeFalsy()
     })
   })
 
   describe('a11y tests', () => {
-    beforeEach(() => {
-      component = render(<Link {...defaultProps} />)
-    })
-
     it('includes a11yLabel', async () => {
-      expect(component.root.props.accessibilityLabel).toBe('a11yLabel override')
+      render(<Link {...defaultProps} />)
+      expect(screen.getByLabelText('a11yLabel override')).toBeOnTheScreen()
     })
 
     it('includes a11yHint', async () => {
-      expect(component.root.props.accessibilityHint).toBe('a11yHint override')
+      render(<Link {...defaultProps} />)
+      expect(screen.getByHintText('a11yHint override')).toBeOnTheScreen()
     })
 
     it('includes a11yValue', async () => {
```

Comment:
Mentioned in our 1:1 but probably not very clearly, I think we want a second a11yValue test for poking at sending a custom one. 
> /** Optional a11yValue for "[position #] of [list total #]" or a custom value descriptive string */
  a11yValue?: a11yValue | string

So basically a test rendering with defaultProps + a11yValue sent as a string (AKA they have a custom formatted string description and didn't want to leverage our translation based on indexes). It was done this way as [accessibilityValue is rather flexible in use conditions](https://reactnative.dev/docs/accessibility#accessibilityvalue).